lerobot/common/logger.py

```diff
@@ -114,6 +114,8 @@ def log_dict(self, d, step, mode="train"):
         assert mode in {"train", "eval"}
         if self._wandb is not None:
             for k, v in d.items():
+                if not isinstance(v, (int, float, str)):
```

Comment:
From the documentation of [wandb log](https://docs.wandb.ai/ref/python/log) it supports logging quite a few helpful types, like single element np arrays or tensors, and even images, plots, or even videos. Restricting it to `int, float, str` seems too restrictive.

Reply:
Good one. I added a warning. The logging might need an overhaul at some point so I think for now this will suffice.